### DISCUSSION TITLE
Add a DQuickItemScissor item

### DISCUSSION THREAD

**Discussion by lychee-yycy** [linuxdeepin/developer-center#8618] [Features Request & Ideas | 特性请求 & 头脑风暴]
It will be more powerful than . Compared with the existing item, 
Support border customization, the following attributes are expected to be added：
- border
  - border.color : color
  - border.width : int

- bottomLeftRadius : real
- bottomRightRadius : real
- topLeftRadius : real
- topRightRadius : real

![DQuickItemScissor vs DQuickItemViewport](https://github.com/linuxdeepin/developer-center/assets/149312566/40ec3bef-6985-45e8-9be1-107057c03f45)

Implemented in the linuxdeepin/dtkdeclarative project

**Comment by kegechen** [comment]
@18202781743
